core/pom.xml

```diff
@@ -319,6 +319,12 @@
       <artifactId>selenium-java</artifactId>
       <scope>test</scope>
     </dependency>
+    <!-- Added for selenium: -->
+    <dependency>
+      <groupId>xml-apis</groupId>
+      <artifactId>xml-apis</artifactId>
+      <scope>test</scope>
```

Comment:
So selenium depends on this but doesn't declare it?

Also, super minor, but the "scope" here is probably redundant.